packages/cyberstorm/package.json

```diff
@@ -17,6 +17,7 @@
     "@babel/runtime": "^7.17.0",
     "@fortawesome/fontawesome-svg-core": "^6.2.0",
     "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/pro-regular-svg-icons": "^6.2.1",
```

Comment:
Mismatching package versions.

Reply:
The mismatching package versions will be corrected by #593 And that PR will be merged before the component PRs

Reply:
Leaving this unresolved for now. It can be marked resolved once #593 is merged.